src/view.cpp

```diff
@@ -359,8 +359,16 @@ void view::open_in_browser(const std::string& url) {
 		cmdline.append("'");
 	}
 	stfl::reset();
-	utils::run_interactively(cmdline, "view::open_in_browser");
+	// store the shell return value
+	int status_browser = utils::run_interactively(cmdline, "view::open_in_browser");
+	// the browser returned a value unequal to zero
+	if (status_browser) {
+		std::clog << strprintf::fmt(_("Starting the browser failed with error code %d"), status_browser) << std::endl;
```

Comment:
Newsbeuter has custom logging facility; grep sources for `LOG` macro. I think it's useful to use `level::DEBUG` here, and show the `cmdline` in addition to its exit code.